cmd/oras/internal/display/status/progress/manager.go

```diff
@@ -120,6 +122,17 @@ func (m *manager) Add() (Status, error) {
 	return m.statusChan(s), nil
 }
 
+func (m *manager) Send(desc ocispec.Descriptor, prompt string) error {
```

Comment:
Missing comment. Also it should be `SendAndStop()`.

Reply:
Done